root/maptick_plot.gnuplot

```diff
@@ -72,40 +80,57 @@ serverfiles = "manuel.csv basil.csv sybil.csv terry.csv"
 
 set output outputfile4
 set title "Average Maptick Over Player Count By Map"
-plot for [file in mapfiles] sprintf("%s%s", inputfolder, file) using 1:7 with lines title file 
+plot for [file in mapfiles] sprintf("%s%s", inputfolder, file) using 1:16 with lines title file 
 
 set output outputfile5
 set title "Average Maptick Over Player Count By Server"
-plot for [file in serverfiles] sprintf("%s%s", inputfolder, file) using 1:7 with lines title file 
+plot for [file in serverfiles] sprintf("%s%s", inputfolder, file) using 1:16 with lines title file 
 
 set ylabel 'Player Count'
 
 set output outputfile6
 set title "Average Player Count"
-plot inputfile using 1:15 with lines
+plot inputfile using 1:22 with lines
 
 set output outputfile7
 set title "Average Player Count By Map"
-plot for [file in mapfiles] sprintf("%s%s", inputfolder, file) using 1:15 with lines title file
+plot for [file in mapfiles] sprintf("%s%s", inputfolder, file) using 1:22 with lines title file
 
 set output outputfile8
 set title "Average Player Count By Server"
-plot for [file in serverfiles] sprintf("%s%s", inputfolder, file) using 1:15 with lines title file
+plot for [file in serverfiles] sprintf("%s%s", inputfolder, file) using 1:22 with lines title file
 
 set ylabel 'TD'
 
 set output outputfile9
 set title "Average TD"
-plot inputfile using 1:16 with lines
+plot inputfile using 1:25 with lines
 
 set output outputfile10
 set title "Average TD By Map"
-plot for [file in mapfiles] sprintf("%s%s", inputfolder, file) using 1:16 with lines title file
+plot for [file in mapfiles] sprintf("%s%s", inputfolder, file) using 1:25 with lines title file
 
 set output outputfile11
 set title "Average TD By Server"
-plot for [file in serverfiles] sprintf("%s%s", inputfolder, file) using 1:16 with lines title file
+plot for [file in serverfiles] sprintf("%s%s", inputfolder, file) using 1:25 with lines title file
 
 set output outputfile12
 set title "TD"
-plot inputfile using 1:5 with lines title "TD"
\ No newline at end of file
+plot inputfile using 1:5 with lines title "TD"
+
+set ylabel 'Microseconds'
+set xrange [163000:]
+
+set output outputfile13
+set title "Average Costs"
+plot inputfile using 1:34 with lines, '' using 1:37 with lines, '' using 1:40 with lines, '' using 1:43 with lines, '' using 1:46 with lines, '' using 1:49 with lines, '' using 1:52 with lines#, '' using 1:55 with lines
```

Comment:
A: Do you want these to show in the legend as their csv keys? They're a bit messy
B: If A, you should increase rmargine so they don't overlap the graph